workspaces/web/src/components/ChallengePanel.svelte

```diff
@@ -73,12 +76,19 @@
     color: $text-invert;
     background: $yellow;
   }
-
-  .panel .right {
+  .left {
+    float: left;
+  }
+  .right {
     float: right;
   }
 
   .panel-block {
     color: inherit;
+    padding-top: 1em;
+    margin: 0 10em;
```

Comment:
I wonder why we need this specific 10em margin here?

Reply:
The margin present here is to put the buttons in a more center position. I used em as its a relative unit.
I followed the design present at https://github.com/kantord/LibreLingo/issues/81#issuecomment-601997442